src/Faker/Container/ContainerBuilder.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Container;
+
+use Faker\Core\File;
+use Faker\Extension\FileExtension;
```

Comment:
@Nyholm 

> Hm..
>
>`Faker\Extension\File` is an interface. `Faker\English\FileExtension` is the implementation of that interface.
>
>I could have named it `Faker\English\File` but that would be more confusing.

It is only confusing when we always import every single class:

```diff
diff --git a/src/Faker/Container/ContainerBuilder.php b/src/Faker/Container/ContainerBuilder.php
index b86992b9..109f03ea 100644
--- a/src/Faker/Container/ContainerBuilder.php
+++ b/src/Faker/Container/ContainerBuilder.php
@@ -4,8 +4,8 @@ declare(strict_types=1);

 namespace Faker\Container;

-use Faker\Core\File;
-use Faker\Extension\FileExtension;
+use Faker\Core;
+use Faker\Extension;
 use Psr\Container\ContainerInterface;

 /**
@@ -61,7 +61,7 @@ final class ContainerBuilder
     public static function defaultExtensions(): array
     {
         return [
-            FileExtension::class => File::class
+            Extension\File::class => Core\File::class
         ];
     }
```